src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PersonControllerTest.kt

```diff
@@ -250,13 +254,17 @@ internal class PersonControllerTest(
           it("returns a 404 status code when a person cannot be found in both upstream APIs") {
             whenever(getPersonService.execute(idThatDoesNotExist)).thenReturn(
               Response(
-                data = null,
+                data = emptyMap(),
```

Comment:
Should this not be a list, as the method signature of this method specifies this to be a list?